lib/spack/llnl/util/tty/log.py

```diff
@@ -853,10 +857,17 @@ def force_echo(self):
         if not self._active:
             raise RuntimeError(
                 "Can't call force_echo() outside log_output region!")
+        # This uses the xon/xoff to highlight regions to be echoed in the
+        # output. We use these control characters rather than, say, a
+        # separate pipe, because they're in-band and assured to appear
+        # exactly before and after the text we want to echo.
+        sys.stdout.write(xon)
```

Comment:
(question) these characters are only handled by `_writer_daemon`, which is only instantiated by `nixlog`, so how is `winlog` making use of this?

Reply:
This was something I was unsure about whether we wanted to leave it in for consistency/later extension or not. They are not currently being used and I'll more than happily remove them.

Reply:
IMO it would be clearest to leave it out of this PR.

Reply:
Agreed. Will do.